src/Service.GraphQLBuilder/Queries/StandardQueryInputs.cs

```diff
@@ -232,7 +231,6 @@ public static InputObjectTypeDefinitionNode UuidInputType() =>
                     new(null, new NameNode("startsWith"), new StringValueNode("Starts With"), new UuidType().ToTypeNode(), null, new List<DirectiveNode>()),
                     new(null, new NameNode("endsWith"), new StringValueNode("Ends With"), new UuidType().ToTypeNode(), null, new List<DirectiveNode>()),
                     new(null, new NameNode("neq"), new StringValueNode("Not Equals"), new UuidType().ToTypeNode(), null, new List<DirectiveNode>()),
-                    new(null, new NameNode("caseInsensitive"), new StringValueNode("Case Insensitive"), new BooleanType().ToTypeNode(), new BooleanValueNode(false), new List<DirectiveNode>()),
```

Comment:
were you able to see which PR merged this change in and perhaps find why the support was not added?

Reply:
This file originally came with https://github.com/Azure/data-api-builder/pull/250 but it isnt clear where Aaron borrowed all of the operators from looking at other modified files.

Reply:
Leveraging git search, it seems like above commit is the one that introduced the string caseInsensitive and it didnt exist before that.
```
Q:\Repos\data-api-builder> git log -S caseInsensitive --oneline
a9d0a622 (HEAD -> caseInsensitive, origin/caseInsensitive) Remove casInsenitive filter operator from schema generated
3176820a Data Api Builder support for multiple databases. (#1709)
8b05b5ba Fix for guid filter in GraphQL (#1644)
a0e9881f Generating a full GraphQL schema from data models (#250)
```